contrib/aws/deis.template.json

```diff
@@ -303,6 +322,68 @@
         ]
       }
     },
+    "DeisPublicWebELB": {
+      "Type": "AWS::ElasticLoadBalancing::LoadBalancer",
+      "Condition": "UsePublicELB",
+      "DependsOn" : "GatewayToInternet",
```

Comment:
Hmm, I thought this needed to be an array of logical names. Okay if this works though.